DIMS-Core/DIMS-Core.DataAccessLayer/Interfaces/IUnitOfWork.cs

```diff
@@ -12,6 +12,6 @@ public interface IUnitOfWork : IDisposable
 
         IReadOnlyRepository<VUserProfile> VUserProfileRepository { get; }
 
-        Task Save();
+        System.Threading.Tasks.Task Save();
```

Comment:
I cannot use aliases, because in this project I have Task class in Models folder and this classes is conficted with System.Threading.Tasks.Task class

Reply:
You can. Just give a different name for System.Threading.Tasks.Task. For example: ThreadingTask